src/main/java/racingcar/DiceRacingRule.java

```diff
@@ -0,0 +1,25 @@
+package racingcar;
+
+import racingcar.race.Dice;
+import racingcar.race.RacingRule;
+
+public class DiceRacingRule implements RacingRule {
+
+    public static final int STOP_MOVE = 0;
+    public static final int MOVE_FORWARD_COUNT = 1;
+
+    private final Dice dice;
+
+    public DiceRacingRule(Dice dice) {
+        this.dice = dice;
+    }
+
+    @Override
+    public int getMoveForward() {
+        int forwardCount = MOVE_FORWARD_COUNT;
+        if(dice.generate() < 4) {
+            forwardCount = STOP_MOVE;
+        }
+        return forwardCount;
```

Comment:
```suggestion
        if(dice.generate() < 4) {
            return STOP_MOVE;
        }
        return MOVE_FORWARD_COUNT;;
```
이렇게 정리할 수 있을 것 같아요:)